src/index.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+;(function() {
+  var stories = new Stories();
+  var storiesList;
+  var aylienUrl = "https://api.aylien.com/api/v1/summarize?url=";
+
+
+  window.addEventListener("load", function(){
+    storiesList = downloadStories();
```

Comment:
This feels like quite a lot to put in an anonymous function. Usually, I'd want to a call back to do one thing - if it's a simple thing like 'make this button green' then it's find for the code just to live in the anonymous function. 

In this case though it seems like what we're doing is 'render the page', so we should delegate all of that function to a class, e.g.`window.addEventListener("load", app.render)` (where app is an instance of ab Application class and render is a method responsible for building the page)